uploaddocs.sh

```diff
@@ -66,12 +66,15 @@ do
     echo "Final upload stage (googleapis.dev)"
     python -m docuploader upload . --credentials $SERVICE_ACCOUNT_JSON --staging-bucket $GOOGLEAPIS_DEV_STAGING_BUCKET
 
-    cd ../devsite/api
+    # For DevSite, we just upload the yaml in the obj/api directory
+    cd ../obj/api
     echo "Generating metadata (DevSite)"
-    python -m docuploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
+    # FIXME: WE DON'T CURRENTLY KNOW API_SHORTNAME
```

Comment:
Looking into this. Not sure the best way to do it yet -- or maybe we should just drop it completely.